src/services/EnvironmentVariables.ts

```diff
@@ -326,8 +326,8 @@ export class EnvironmentVariables {
 		return this.RESOURCES_TTL_SECS;
 	}
 
-	clientSessionTokenTtlInSeconds(): any {
-		return this.CLIENT_SESSION_TOKEN_TTL_SECS;
+	clientSessionTokenTtlInDays(): any {
```

Comment:
number instead of any, same for other fields that are expected to return number

Reply:
Done